### ISSUE TITLE
[FEATURE]Message abspeichern bzw. automatisch abspeichern zur Akte

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-org#2339]
Ich freunde mich gerade mit dem Messanger an (statt Notizzettel). Ein feature finde ich nicht / fehlt mir. Eine Nachricht, die ich bestätige (automatisch) zur Akte oder zu einer Akte nehmen zu können.

**Comment by j-dimension** [comment]
Wenn die Nachricht einen Aktenbezug hat, dann steht sie immer sichtbar auf dem ersten Tab der Akte.

**Comment by Mucki526** [comment]
Danke, das macht mich zu 25% glücklicher :-). Aber ich kann diese Nachricht nicht in den Dokumentenordner übernehmen. Es muss also immer eine Notiz geschrieben und mit einer Message vesendet werden im Moment, Und die Message kann ich dann nicht übernehmen ah,,, kommt in 2.7...

**Comment by j-dimension** [comment]
Warum als Dokument - die Daten sind doch auf dem ersten Tab. Oder geht es um die Auffindbarkeit per Volltextsuche?

**Comment by Linkki2021** [comment]
Ich vermute, es geht um die Gesamtübesichtlichkeit (alles an einem Ort). Ich habe aus der operativen Fallbearbeitung ähnliches Feedback, dass man die Notizen gerne gemeinsam im zeitlichen Verlauf sortiert mit den Dokumenten sehen würde.

**Comment by Mucki526** [comment]
Richtig, es geht um den zeitlichen Ablauf, vor allem, wenn ein anderer Anwalt sich durch die Akte lesen muss und links das Telefon am Ohr hat, während er rechts durchblättert. Gesamtübersichtlichkeit und workflow.....

**Comment by Mucki526** [comment]
Genau genommen sind es zwei Sichtweisen: Ich bekomme eine Notiz und will sie in eine Akte ablegen (oder sie kommt zur Akte) und ich will sie ablegen. 
Auf der anderen Seite. Meine Kollegin ist nicht da. Ihr Mdt(in) ruft an. Ich nehme das Problem entgegen und schreibe darüber eine Notiz, die muss ich dann ausdrucken oder als "Fresszettel" auf die Tastatur legen (Telefonnotiz). Als Message, die ich gleichzeitig (soweit ich berechtigt bin) gleichzeitig in die Akte lege, ist beides ohne "Zettelkram" erledigt. Notiz in Akte in Form der Nachricht. Und ich spare mir 1000 Notizzettel im Jahr....

**Comment by Linkki2021** [comment]
> Also eine Aktion auf eine Nachricht, welche die Nachricht in eine Notiz wandelt und als Dokument ablegt?

Meines Erachtens gibt es zwei grundsätzliche Umsetzungsoptionen:
a) Wie von Dir beschrieben: man wählt eine Message aus, die dann spezifisch als Dokument abgelegt wird.
b) Jede Message wird automatisch in die Dokumente "gespiegelt". Wahlweise könnte man dort auswählen, ob man Messages angezeigt haben möchte, oder nicht.

Gibt sicher für beides Für und Wider, und eine Frage des Umsetzungsaufwands ist es vermutlich auch. Wir wären das außen vor gelassen eher bei b)
